make_prg/subcommands/from_msa.py

```diff
@@ -1,23 +1,46 @@
-import logging
+from typing import List
+import multiprocessing
 from pathlib import Path
-
-from make_prg.from_msa import prg_builder, NESTING_LVL, MIN_MATCH_LEN
-from make_prg import io_utils
-from make_prg.subcommands.output_type import OutputType
+from loguru import logger
+from make_prg import prg_builder
+from make_prg.from_msa import NESTING_LVL, MIN_MATCH_LEN
+from make_prg.utils import io_utils, gfa
 
 
 def register_parser(subparsers):
     subparser_msa = subparsers.add_parser(
         "from_msa",
-        usage="make_prg from_msa [options] <MSA input file>",
+        usage="make_prg from_msa",
         help="Make PRG from multiple sequence alignment",
     )
-
     subparser_msa.add_argument(
-        "MSA",
+        "-i",
+        "--input",
         action="store",
         type=str,
-        help="Input file: a multiple sequence alignment",
+        required=True,
+        help=(
+            "Input data. If it is a file, will be read as the supported alignment_format (see "
+            "parameter --alignment_format). A single PRG, GFA and update data structures will "
+            "be created for the given MSA. If it is a dir, will process all files in the dir. "
+            "Multiple PRGs, GFAs and update data structures will be created, one for each MSA"
+        ),
+    )
+    subparser_msa.add_argument(
+        "-o",
+        "--output_prefix",
```

Comment:
```suggestion
        "--output-prefix",
```
I don't know why, but I have a real dislike for underscores in CLI options. This is the height of fussiness I know, but I get very particular about CLIs. If someone else has a strong opinion *for* underscores, then leave it as is.

If we change this, I guess we have to change the other options with underscores for uniformity.

Reply:
Will change this and all the other options from scores to dashes for uniformity

Reply:
Done in https://github.com/iqbal-lab-org/make_prg/pull/35/commits/eee33ad6f266062c34a8f0918504cf0cb4a1793e